web/js/map/natural-events/cluster.js

```diff
@@ -93,6 +95,8 @@ export const getClusters = ({ geometry, id }, proj, selectedDate, map) => {
   const selectedCoords = geometry.find(({ date }) => date.split('T')[0] === selectedDate).coordinates;
   geometry.forEach((geom) => {
     let { coordinates } = geom;
+    const { magnitudeUnit, magnitudeValue } = geom;
```

Comment:
This is a nitpick but, it seems somewhat redundant to destructure two props off an object just to make a new object with those same two props.  I think it's cleaner to either just pass the `geom` object to the `clusterPointToGeoJson` function and destructure the props you need in the param definition of that function signature (which you're already doing).

Or, destructure them here like you are and just pass them as separate parameters.

Reply:
Ah yeah.  When I was working this out, it made sense to do it this way.  Definitely makes sense to just pass the `geom` object and destructure from that. Thanks!